src/Message/AbstractRequest.php

```diff
@@ -1,217 +1,217 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Omnipay\WestpacPaywayRest\Message;
 
+use Money\Money;
+use Omnipay\Common\Message\ResponseInterface;
+
 /**
  * @link https://www.payway.com.au/rest-docs/index.html
  */
 abstract class AbstractRequest extends \Omnipay\Common\Message\AbstractRequest
 {
-    /** @var string Endpoint URL */
-    protected $endpoint = 'https://api.payway.com.au/rest/v1';
+    protected string $endpoint = 'https://api.payway.com.au/rest/v1';
 
-    abstract public function getEndpoint();
+    abstract public function getEndpoint(): string;
 
     /**
      * Get API publishable key
-     * @return string
      */
-    public function getApiKeyPublic()
+    public function getApiKeyPublic(): ?string
     {
         return $this->getParameter('apiKeyPublic');
     }
 
     /**
      * Set API publishable key
-     * @param  string $value API publishable key
      */
-    public function setApiKeyPublic($value)
+    public function setApiKeyPublic(string $value): self
     {
         return $this->setParameter('apiKeyPublic', $value);
     }
 
     /**
      * Get API secret key
-     * @return string
      */
-    public function getApiKeySecret()
+    public function getApiKeySecret(): ?string
     {
         return $this->getParameter('apiKeySecret');
     }
 
     /**
      * Set API secret key
-     * @param  string $value API secret key
      */
-    public function setApiKeySecret($value)
+    public function setApiKeySecret(string $value): self
     {
         return $this->setParameter('apiKeySecret', $value);
     }
 
     /**
      * Get Merchant
-     * @return string Merchant ID
      */
-    public function getMerchantId()
+    public function getMerchantId(): ?string
     {
         return $this->getParameter('merchantId');
     }
 
     /**
      * Set Merchant
-     * @param  string $value Merchant ID
      */
-    public function setMerchantId($value)
+    public function setMerchantId(string $value): self
     {
         return $this->setParameter('merchantId', $value);
     }
 
     /**
      * Get Use Secret Key setting
-     * @return bool Use secret API key if true
      */
-    public function getUseSecretKey()
+    public function getUseSecretKey(): ?bool
     {
-        return $this->getParameter('useSecretKey');
+        return (bool) $this->getParameter('useSecretKey');
     }
 
     /**
      * Set Use Secret Key setting
-     * @param  string $value Flag to use secret key
      */
-    public function setUseSecretKey($value)
+    public function setUseSecretKey(string $value): self
```

Comment:
Do values have to be strings to these methods? seems weird to pass a string to set a bool

Reply:
Seems we don't use it in any of our code.  I type-hinted string because that's what it was annotated as.  I'll change it to string|bool just in case.